docker/configure_workers_and_start.py

```diff
@@ -770,7 +794,15 @@ def generate_worker_files(
 
     # A list of internal endpoints to healthcheck, starting with the main process
     # which exists even if no workers do.
-    healthcheck_urls = ["http://localhost:8080/health"]
+    # This list ends up being part of the command line to curl, which added unix socket
+    # support in version 7.40. The scheme and hostname are ignored, the path is not.
```

Comment:
My first question was "version of what" but just took a second look to understand. Maybe more clear:

```suggestion
    # This list ends up being part of the command line to curl, (curl added support unix sockets
    # in version 7.40).
```